src/Stormpath/Resource/Application.php

```diff
@@ -413,6 +413,36 @@ public function getAccount(ProviderAccountRequest $request)
             $providerAccountAccess, Stormpath::PROVIDER_ACCOUNT_RESULT);
     }
 
+    public function sendVerificationEmail(VerificationEmailRequest $request)
+    {
+        if ($request == null)
+        {
+            throw new \InvalidArgumentException('VerificationEmailRequest cannot be null');
+        }
+
+        $login = $request->getLogin();
+        if ($login == null or ($login != null and $login == ''))
+        {
+            throw new \InvalidArgumentException('VerificationEmailRequest\'s login property is required');
+        }
+
+        $accountStore = $request->getAccountStore();
+        if ($accountStore != null && $accountStore->href == null)
+        {
+            throw new \InvalidArgumentException("verificationEmailRequest's accountStore has been specified but its href is null.");
+        }
+
+        $verificationEmail = VerificationEmails::instantiate();
```

Comment:
Please use the the data store for instantiating as the user might not be using the static client config. Also, I don't think a user will ever need to instantiate a `VerificationEmail` resource class, unless I'm missing something; so that method could be removed.